docs/guide-metadata.md

```diff
@@ -174,7 +174,45 @@ build --workspace_status_command=$(pwd)/workspace_status.sh
 
 Then you'll need to add a `workspace_status.sh` file to the root of your workspace that prints `HOST yourhost`.
 
-### Custom Links
+## Pattern
+
+By default a build's pattern is determined by bazel command that is run.
+
+If the bazel command is:
+
+```
+bazel build //...
+```
+
+The pattern would be:
+
+```
+//...
+```
+
+You can override this using build metadata or workspace info.
```

Comment:
Should we add an example here and maybe say something like:

This can be used for more advanced use cases, such as a tool that generates a `bazel build` command with a series of patterns, but wants to provide a more user-friendly version that is equivalent.

Reply:
Added a note on it being useful for more advanced use-cases.